### ISSUE TITLE
Accessibility - HTML improvements

### ISSUE THREAD

**Issue by kaylaondracek** [mlibrary/fishrappr#175]
## Description: 

Implement CSS/HTML changes noted by Colin during accessibility review where allowable by Bootstrap.

**modified to remove CSS-related ACs, added to #194**

Accessibility review:
https://docs.google.com/document/d/15WNvjeG9KFSl0PIqLsASlAyNnGIHFwHfriLzs052nWQ/edit

_Gordon: I added the complete text of Colin's initial review (minus some CSS items) below in comments._

## Acceptance Criteria:
Home page
- [x] Add date filter dropdown labels (hidden) to home page and search page
- [x] Change "read more" link to read "about" or similar - UX 
- [x] Wrap `<a>` tags around images and date in "Today..." section
- [x] Wrap items in "Today..." section with `<figure>` element with `<figcaption>` text instead of `<h3>`

Image viewer
- [ ] ensure focus remains on zoom in/out buttons (currently focus moves to the newspaper image after each click
- [ ] increase contrast for zoom & print buttons
- [x] remove `<a>` from date/issue number text at top of page
- [ ] Inside the focusable element for the image viewer should be a `<p>` tag with text like "scanned image of the page"

**Comment by gordonleacock** [comment]
Colin Fulton's Notes:
General
There are multiple layout bugs at smaller window sizes across the website.
All pages need a min-width.
The smallest text size should render to 14px at the most (preferably 16px) when the browser is zoomed at 100% (currently there is some text as small as 12px).
Em's need to be be used instead of px for the font-size property.
Many of the linked thumbnails throughout the website lack alt text.
HTML5 sectioning elements need to be used to illustrate the semantic structure of each page.
The focus style for links, form elements, etc should be defined because the default focus style in OSX isn't high enough contrast for some elements.
The search form should have aria-live="polite"

**Comment by gordonleacock** [comment]
Colin Fulton's Notes:

**Home**

- [ ] Date filter dropdowns lack labels (they can be hidden labels).
- [ ] "Read More" links need better text (the text of a link should describe what it goes to, so no "click here" or "read more").
- [ ] There is no <h1> tag to say what page the user is on (this doesn't have to be visible, but the "Michigan Daily Archive" header would make a good <h1> for the home page).
- [ ] The <a> tags in the "Today in the Michigan Daily section" should wrap around the images (which should have alt="") and the date text so the whole thing operates as one link, rather than two distinct links that go to the same place.
- [ ] Each item in the "Today in the Michigan Daily" section should be wrapped in a <figure> element with the text in a <figcaption> instead of an <h3> (since it isn't a heading for anything).

**Comment by gordonleacock** [comment]
Colin Fulton's Notes:

**Search Results**

- [ ] The fallback color for highlighted text isn't necessary (we don't support any browsers which don't allow for rgba() colors), especially since the fallback is too low contrast.
- [ ] Each search result should be in an <article> instead of a <div>.
- [ ] The facet headers that act as show/hide buttons should be <button> elements instead of <a> elements (since the do not link to anywhere).
- [ ] The linked thumbnail in the search results needs alt text to describe where the link goes.
- [ ] The <div> elements that are highlighting the text should either have role="presentation" added to them or they should be drawn via <rect> elements inside an <svg> element with role="presentation"
- [ ] The "[remove]" buttons for filters should describe what they are removing in the screen-reader-only text.
- [ ] There should be some screen-reader-only text to describe what the numbers next to each filter means. Something like "42 records".

**Comment by gordonleacock** [comment]
Colin Fulton's Notes:

**Records**

- [ ] The highlights over the text are a bit too low contrast. Either only outline the highlighted text or (with the current color) reduce the fill-opacity to 0.15 at the most.
- [ ] When clicking the Zoom In/Out buttons, focus should remain on the zoom buttons. Currently focus is moved to the newspaper image after each click.
- [ ] Visible instructions or a link to instructions should be on each record page to instruct users that they can pan around the image either by clicking/dragging or moving focus onto the image and using the arrow keys. Heidi would be a good person to write up some short, clear directions.
- [ ] The focus state for the Zoom/Print buttons should still have proper contrast. Currently the text goes to black and the contrast becomes too low.
- [ ] The "Copy Link" button does not work, and the permalink text can be edited by the user. (Tested using Safari 9.)
- [ ] The date/issue number text at the top of the page shouldn't be a link (it isn't linked to anywhere).
- [ ] Each of the image-tiles inside the viewer needs alt="" as well as role="presentation" and aria-hidden="true".
- [ ] Inside the focusable element for the image viewer should be a <p> tag with text like "scanned image of the page". (Focusable elements can't be hidden from most assistive technologies so there needs to be some text to say what the image viewer is.)
